### ISSUE TITLE
About transformer prediction results

### ISSUE THREAD

**Issue by ChaoyueDing** [LongxingTan/Time-series-prediction#2]
I hope to get your suggestions on how to solve the problem about transformer prediction. I used the transformer model in the code to predict the data and got the results shown in the following figure. The transformer predicted results are horizontal and cannot fit the original upward sloping data.

![transformer](https://user-images.githubusercontent.com/38074924/82110093-5c5cbe00-976e-11ea-9565-0d169ce7d68b.png)

**Comment by sauradefy99** [comment]
Just a guess but looks like the transformer is trying to predict the trend-removed data?

**Comment by ChaoyueDing** [comment]
May be so, thank you for your reply.

**Comment by ChaoyueDing** [comment]
May be so, thank you for your reply.

**Comment by deshanej** [comment]
I am also getting 'constant' values for predictions (ie no time variation,)anybody had any luck with a good prediction?

**Comment by helloworld-wu** [comment]
hello, did you solved this problem?

**Comment by helloworld-wu** [comment]
I am also getting 'constant' values for predictions (ie no time variation)

**Comment by djph29** [comment]
Yes, increasing the number of epochs, and changing the input_sequence_length helped.

**Comment by MyLinChi** [comment]
I have this same problem too.And increasing the number of epochs or changing the input_sequence_length doesn't work.
